### DISCUSSION TITLE
[SUGGESTION] Glass Heptas and Tetras

### DISCUSSION THREAD

**Discussion by TIKIRobo** [tsunamayo/Starship-EVO#3972] [Suggestions]
i know i know, you've heard it a million times but the current glass tetras are trash, especially when compared to the new glass blocks, all we need to finally begin making some sexy glass cockpits are just glass blocks in the shape of the normal brick tetra and brick hepta, they would also stretch similarly to the normal brick versions but ofc be transparent, this would allow me and other builders to create smoother and nicer cockpit windows 

![Starship EVO 4_12_2021 6_44_33 PM](https://user-images.githubusercontent.com/56371294/114484477-25ca3600-9bbf-11eb-8588-e5c929a0a4ea.png)
this cockpit looks ok, but it would be a lot better if i could make the side parts out of glass 

Thank you for your time
-TIKI

**Comment by ProPeach** [comment]
#2987 for a handy visual ; )

**Comment by KINGrimson** [comment]
yes please

**Comment by tsunamayo** [comment]
Why are they trash? thx

**Comment by Kaiser-Indrasil** [comment]
Simply because of their rectangular hitbox, which forces people to place them on child entities and move them away for the time they build the detailing. This detailing should be easily placeable next to the Tetra Glass because it doesn't clip into the glass visually, but the hitbox prevents it. I've encountered this problem when working on my [Agni fighter](https://steamcommunity.com/sharedfiles/filedetails/?id=2297275555). Super annoying.

**Comment by BigBadKangaroo** [comment]
I'd love to give it a smooth shape, but the block itself is not placeable in a way that you can hide the borders. And placing just one block is not possible because of the hitbox on that thing!
![20210413132837_1](https://user-images.githubusercontent.com/81495956/114545388-3086d880-9c5c-11eb-9668-38b27450b7d5.jpg)

Would love a much smoother looking block - but I don't know how this would be / if it's even possible

**Comment by TIKIRobo** [comment]
> Why are they trash? thx

like what others have said before, they have some very odd hitboxes but there is also the problem of their size, they are too big and stretch too weirdly to work well in more "smooth" shapes, if they were .25x.25x.25 instead of .5x.5x.5 that would help a lot

**Comment by BigBadKangaroo** [comment]
> 
> 
> > Why are they trash? thx
> 
> like what others have said before, they have some very odd hitboxes but there is also the problem of their size, they are too big and stretch too weirdly to work well in more "smooth" shapes, if they were .25x.25x.25 instead of .5x.5x.5 that would help a lot

Just overhauling the hitbox of it would make it 10x more useful imo.

**Comment by TIKIRobo** [comment]
> Just overhauling the hitbox of it would make it 10x more useful imo.

that would help yes but unfortunately they tend to be a little to large for most fighters so there should also be a version thats .25^3
